test_fixtures/require_test/file.gb

```diff
@@ -0,0 +1,6 @@
+require("file")
```

Comment:
You should also test these methods in separate files. And test them like I test `require_relative`  in [here](https://github.com/goby-lang/goby/blob/master/goby_test.go). You should execute these files and check if the return value is right.